### ISSUE TITLE
Add a return parameter for the current location for the onLongPress of the DataRow of the dataTable

### ISSUE THREAD

**Issue by laterdayi** [flutter/flutter#151806]
### Use case

Add a return parameter for the current location for the onLongPress of the DataRow of the dataTable

ref https://github.com/flutter/flutter/issues/151745

### Proposal

Add a return parameter for the current location for the onLongPress of the DataRow of the dataTable

**Comment by darshankawar** [comment]
@laterdayi 
Check if https://api.flutter.dev/flutter/gestures/DragUpdateDetails/localPosition.html helps in your case or not.

**Comment by laterdayi** [comment]
> @laterdayi Check if https://api.flutter.dev/flutter/gestures/DragUpdateDetails/localPosition.html helps in your case or not.

Yes, just like onDrag, is onLongPress of the DataRow of the dataTable available now? How should I use it

**Comment by darshankawar** [comment]
See if [this](https://stackoverflow.com/a/68031160) helps in your case or not.

**Comment by laterdayi** [comment]
> See if [this](https://stackoverflow.com/a/68031160) helps in your case or not.

This is just set in DataCell for one quick, I need to click on the set for the entire row
